src/common/url-validator.ts

```diff
@@ -6,14 +6,19 @@ export class UrlValidator {
     public async isSupportedUrl(url: string, chromeAdapter: BrowserAdapter): Promise<boolean> {
         const lowerCasedUrl: string = url.toLowerCase();
         if (lowerCasedUrl.match('http://*/*') || lowerCasedUrl.match('https://*/*')) {
-            return lowerCasedUrl.indexOf('https://chrome.google.com') !== 0;
+            return this.hasSupportedPrefix(lowerCasedUrl);
         } else if (UrlValidator.isFileUrl(lowerCasedUrl)) {
             return await this.checkAccessToFileUrl(chromeAdapter);
         } else {
             return false;
         }
     }
 
+    private hasSupportedPrefix(lowerCasedUrl: string): boolean {
+        const unsupportedPrefixes = ['https://chrome.google.com/', 'https://microsoftedge.microsoft.com/'];
```

Comment:
Currently on master/production we are able to scan https://microsoftedge.microsoft.com using our chrome extension. This change means we won't. The opposite is also true: we won't be able to scan urls that contains 'chrome' using the edge extension. Is that the correct intention? This sounds like one of several behaviors we should consider (and implement) as differential behavior between chrome and edge extension.

Reply:
It is true that Chrome can currently scan https://microsoftedge.microsoft.com and would not be able to after this change. Edge Insider, on the other hand, is prevented from scanning both Chrome Web Store pages and Edge Insider Addon pages. I think things will be simpler if we can avoid differential behavior between the two browsers if at all possible.